src/index.html

```diff
@@ -18,11 +18,11 @@ <h2 class="recent-scores">Recent scores</h2>
     <div class="score-heading">
       <h2 class="add-score">Add your score</h2>
     </div>
-    <form action="#" class="player-info">
+    <form action="#" class="player-info" method="POST">
       <input type="text" id="name" placeholder="Your name" />
       <input type="text" id="score" placeholder="Your score" />
       <div class="submit-button">
```

Comment:
- [ optional] Great job. your board works perfectly with the endpoints.  you can set your inputs to reset(clear) when you click the submit button.

Reply:
Thank you @tufoinnkuo10 . I have been looking for a way to do that. I tried setting the values to "" after the javascript click event was over but it gave  me errors. Any suggestion?